ios/ReactTestApp.xcodeproj/project.pbxproj

```diff
@@ -220,8 +220,8 @@
 		19ECD0CA232ED425003D8557 /* Project object */ = {
 			isa = PBXProject;
 			attributes = {
-				LastSwiftUpdateCheck = 1100;
-				LastUpgradeCheck = 1220;
+				LastSwiftUpdateCheck = 1230;
+				LastUpgradeCheck = 1230;
```

Comment:
Are these changes happening because you are using Xcode 12.3? Not relevant per se, just wondering

Reply:
Yeah, I had just installed Xcode 12.3 😄